src/components/Mission.jsx

```diff
@@ -0,0 +1,49 @@
+import PropTypes from 'prop-types';
+
+const MemberSpan = ({ activeMember }) => {
+  return activeMember ? (
+    <span className="text-xs text-white font-bold whitespace-nowrap bg-cyan-600 px-2 py-1 rounded-md">Active Member</span>
+  ) : (
+    <span className="text-xs text-white font-bold whitespace-nowrap bg-gray-500 px-2 py-1 rounded-md">Not a member</span>
+  );
+};
+
+const ActionButton = ({ activeMember }) => {
+  return activeMember ? (
+    <button className="px-3 py-2 rounded-md border text-red-600 border-red-600">Leave Mission</button>
+  ) : (
+    <button className="px-3 py-1 rounded-md border text-gray-600 border-gray-600">Join Mission</button>
+  )
+}
+
+function Mission({ name, desc, dark, activeMember }) {
+  return (
+    <tr className={dark ? "bg-gray-200" : ""}>
```

Comment:
- [ ] [OPTIONAL] Instead of using a dark prop, you can use the property of css `:nth-child(odd)` to the parent element and set a bg gray for every odd child element. 😎